.github/workflows/haskell.yml

```diff
@@ -75,6 +98,22 @@ jobs:
       with:
         use-sodium-vrf: true # default is true
 
+    - name: "Re-install openssl on Windows"
+      if: runner.os == 'Windows'
+      shell: C:/msys64/usr/bin/bash.exe -e '{0}'
+      env:
+        MSYSTEM: MINGW64
+        # do not ever change to $HOME by yourself.
+        CHERE_INVOKING: 1
+        # do we want to inherit the path?
+        # MSYS2_PATH_TYPE: inherit
+      run: |
+        /usr/bin/pacman --noconfirm -R mingw-w64-x86_64-openssl
+
+        curl -O https://repo.msys2.org/mingw/mingw64/mingw-w64-x86_64-openssl-3.2.1-1-any.pkg.tar.zst
```

Comment:
Can you link to somewhere explaining why this specific version is being chosen? Maybe the Slack thread with @angerman would be a good pointer.

Reply:
These are installed as part of the updated action in https://github.com/input-output-hk/actions/pull/22. But the issue for windows is blst, we might be able to downgrade that to fix windows.